.github/workflows/test-enos-scenario-ui.yml

```diff
@@ -100,15 +99,16 @@ jobs:
         if: contains(${{ github.event.repository.name }}, 'ent')
         run: echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
       - name: Install Chrome Dependencies
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
```

Comment:
Do we want to also add the condition to check if chrome is already installed incase of re-run ?

Reply:
yeah, good point... originally I had a check that checked if chrome was installed and it didn't seem to work. I can take a moment to revisit that approach and see if I can get it to work.

Reply:
updated to use a check that installs chrome if chrome is not installed.